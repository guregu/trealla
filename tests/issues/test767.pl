:- initialization(main).

main :-
	number_chars(N,"0'\\0\\"), write(N), nl,
	X = 0'\12\, write(X), nl.

