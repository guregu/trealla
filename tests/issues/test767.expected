   Warning: Initialization goal exception: error(syntax_error(number),number_chars/2).
